### PR TITLE
Add Fork 1.37.2

### PR DESCRIPTION
**Pull request by Velgus**
Issue #1094

### CODE REVIEW

**Comment by linsui** [comment]
Please  use one of `Env:LocalAppdate` and `env:LOCALAPPDATA` in both of the two place.

**Comment by Velgus** [comment]
Couldn't Rebase as for some reason `grepwin-install-context.reg` and `grepwin-uninstall-context.reg` are unstaged changes, even though they're identical 😕

**Comment by linsui** [comment]
Same for me...Strange.

File: bucket/fork.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "homepage": "https://git-fork.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://git-fork.com/license"
+    },
+    "url": "https://fork.dev/update/win/Fork-1.37.2-full.nupkg",
+    "hash": "sha1:abaa753d9cbeeeb157ec9ba7b5865bf7a2979986",
+    "extract_dir": "lib\\net45",
+    "post_install": [
+        "$gitVersion = (Get-Item \"$dir\\PortableGit*\") -Replace '[^\\d]+([\\d.]+).+', '$1'",
```
Comment (linsui) [inline bucket/fork.json]:
```suggestion
        "$gitVersion = (Get-Item \"$dir\\PortableGit*\") -Replace '.*-([\\d.]+)-64-bit\\.7z$', '$1'",
```

This is more strict and should also work in installer.

File: bucket/fork.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        "identifier": "Freeware",
+        "url": "https://git-fork.com/license"
+    },
+    "url": "https://fork.dev/update/win/Fork-1.37.2-full.nupkg",
+    "hash": "sha1:abaa753d9cbeeeb157ec9ba7b5865bf7a2979986",
+    "extract_dir": "lib\\net45",
+    "post_install": [
+        "$gitVersion = (Get-Item \"$dir\\PortableGit*\") -Replace '.*-([\\d.]+)-64-bit\\.7z$', '$1'",
+        "New-Item \"$env:LOCALAPPDATA\\Fork\\gitInstance\\$gitVersion\" -ItemType Junction -Target \"$(scoop which git)\\..\\..\" -Force | Out-Null",
+        "Remove-Item \"$dir\\PortableGit-$gitVersion-64-bit.7z\", \"$dir\\lib\", \"$dir\\7z.*\", \"$dir\\Fork_ExecutionStub.exe\" -Force"
```
Comment (Ash258) [inline bucket/fork.json]:
```suggestion
        "@('Fork_ExecutionStub.exe', 'PortableGit-*.7z', 'lib', '7z.*') | ForEach-Object { Remove-Item "$dir\\$_" -Recurse }"
```

**Pull request merged**